ui/desktop/config/environment.js

```diff
@@ -85,6 +86,7 @@ module.exports = function (environment) {
       : true;
 
     // Enable features in development
+    ENV.featureFlags['post-connection-workflow'] = true;
```

Comment:
@cameronperera Shouldn't we include the flag within test environment as well?

Reply:
We did in the past but we decided to move to enabling a feature flag in the test that directly needs it to be enabled.

Reply:
@cameronperera that is totally correct and I completly forget about it. Thanks for the reminder!